test/distrib/cpp/run_distrib_test_cmake.bat

```diff
@@ -29,44 +29,51 @@ powershell -Command "Add-Type -Assembly 'System.IO.Compression.FileSystem'; [Sys
 @rem set absolute path to OpenSSL with forward slashes
 set OPENSSL_DIR=%cd:\=/%/OpenSSL-Win32
 
-cd third_party/zlib
-mkdir cmake
-cd cmake
-cmake -DCMAKE_INSTALL_PREFIX=%INSTALL_DIR% ..
+@rem Install c-ares
+mkdir third_party\cares\cares\cmake\build
+pushd third_party\cares\cares\cmake\build
+cmake -DCMAKE_INSTALL_PREFIX=%INSTALL_DIR% ..\..
 cmake --build . --config Release --target install || goto :error
-cd ../../..
+popd
 
-cd third_party/protobuf/cmake
-mkdir build
-cd build
+@rem Install protobuf
+mkdir third_party\protobuf\cmake\build
+pushd third_party\protobuf\cmake\build
 cmake -DCMAKE_INSTALL_PREFIX=%INSTALL_DIR% -DZLIB_ROOT=%INSTALL_DIR% -Dprotobuf_MSVC_STATIC_RUNTIME=OFF -Dprotobuf_BUILD_TESTS=OFF ..
 cmake --build . --config Release --target install || goto :error
-cd ../../../..
+popd
 
-cd third_party/cares/cares
-mkdir cmake
-cd cmake
-cmake -DCMAKE_INSTALL_PREFIX=%INSTALL_DIR% ..
+@rem Install zlib
+mkdir third_party\zlib\cmake\build
+pushd third_party\zlib\cmake\build
+cmake -DCMAKE_INSTALL_PREFIX=%INSTALL_DIR% ..\..
 cmake --build . --config Release --target install || goto :error
-cd ../../../..
+popd
 
-@rem OpenSSL-Win32 and OpenSSL-Win64 can be downloaded from https://slproweb.com/products/Win32OpenSSL.html
```

Comment:
since we are already updating these scripts, would it make sense to a snippet that will wipe out the submodules before installing gRPC, to make sure we are not depending on them?

on linux we do this:
```
# Just before installing gRPC, wipe out contents of all the submodules to simulate
# a standalone build from an archive
# shellcheck disable=SC2016
git submodule foreach 'cd $toplevel; rm -rf $name'
```

Reply:
Done!